src/orquestra/integrations/qiskit/backend/_qiskit_wavefunction_simulator.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional, List
+
+import numpy as np
+from qiskit import QuantumCircuit, execute
+from qiskit.transpiler import CouplingMap
+from qiskit_aer.noise import NoiseModel
+
+from orquestra.integrations.qiskit.backend._qiskit_runner import AnyQiskitBackend
+from orquestra.integrations.qiskit.conversions import export_to_qiskit
+from orquestra.quantum.api.backend import StateVector
+from orquestra.quantum.api.wavefunction_simulator import BaseWavefunctionSimulator
+from orquestra.quantum.circuits import Circuit
+from orquestra.quantum.circuits.layouts import CircuitConnectivity
```

Comment:
Could you remove `CircuitConnectivity` from `from orquestra.quantum.circuits` and replace it with native qiskit types in here?

Reply:
Fixed in [1ce3987](https://github.com/zapatacomputing/orquestra-qiskit/pull/25/commits/1ce398775ed7588717c6f51aa89ecadf72da47d4)